content/5_Plat/Binary_Jump.mdx

```diff
@@ -129,6 +129,41 @@ int main() {
 
 </CPPSection>
 
+<PySection>
+
+```py
+import math
+
+
+n, q = map(int, input().split())
+
+
+def jmp(x: int, k: int) -> int:  # Get the k'th ancestor of x
```

Comment:
```suggestion
def jmp(x: int, k: int) -> int:
	""":return the kth ancestor of x"""
```
also rename this to kth_ancestor or kth_parent

Reply:
I renamed it.